cli/tests/test/doc_markdown/doc.md

```diff
@@ -0,0 +1,19 @@
+# Documentation
+
+The following block does not have a language attribute and should be ignored:
+
+```
+This is a fenced block without attributes, it's invalid and it should be ignored.
+```
+
+The following block should be given a js extension on extraction:
+
+```js
+console.log("js");
+```
+
+The following block should be given a ts extension on extraction:
+
+```ts
+console.log("ts");
+```
```

Comment:
For the sake of completeness could you add an example that throws an error to showcase how it would look like